File: nomad-samples/admin/js/basic-live-channel/live-operator/stop-broadcast.js
import * as prjConstants from "../constants/project-constants.js";
import apiExceptionHandler from "../exceptions/api-exception-handler.js";

export default async function stopBroadcast(AUTH_TOKEN, ID) {
    // Create header for the request
    const HEADERS = new Headers();
    HEADERS.append("Authorization", `Bearer ${AUTH_TOKEN}`);
    HEADERS.append("Content-Type", "application/json");

    // Send the request
    const RESPONSE = await fetch(`${prjConstants.ADMIN_API_URL}/admin/liveOperator/${ID}/stop`, {
        method: "POST",
        headers: HEADERS
    });

    // Check for success
    if (RESPONSE && RESPONSE.ok) {
        return;
    }

    await apiExceptionHandler(RESPONSE, `Stopping Broadcast ${ID} failed`);
}

File: nomad-samples/admin/py/basic-live-channel-1.1/live_operator/stop_broadcast.py
from constants.project_constants import *
from exceptions.api_exception_handler import *

import requests

def stop_broadcast(AUTH_TOKEN, ID):

    HEADERS = {
        "Authorization": "Bearer " +  AUTH_TOKEN,
        "Content-Type": "application/json"
    }

    try:
        RESPONSE = requests.post(f"{ADMIN_URL}/admin/liveOperator/{ID}/stop", headers= HEADERS)

        # If not found return None
        if (not RESPONSE.ok):
            raise Exception()
    
    except:
        api_exception_handler(RESPONSE, f"Stopping broadcast for Live Channel {ID} failed")